tests/test_resources/test_modules/test_folder.py

```diff
@@ -182,49 +163,23 @@ def test_cluster_and_gcs(self, cluster, cluster_folder):
                 f"on the cluster {cluster.name}. For now please manually enable them directly on the cluster. "
                 f"See https://cloud.google.com/sdk/gcloud/reference/auth/login"
             )
+        
+    @pytest.mark.level("local") # TODO: Needs S3 and GCP credentials
+    def test_folder_and_folder(self, folder):
+        # from one cloud folder to another cloud folder
```

Comment:
need to call `.to()` in order for this test to do anything. e.g. `new_folder = folder.to(system=folder.system)`

Reply:
so if you look at the original `test_s3_and_s3` test (pasted below), the point is to send it to a new folder but on the same system, and test that the contents remain the same. this is different than just testing that the contents exist in the original fixture (which exist on s3)

that being said, I think we can just get rid of this test. `test_cloudA_and_cloudB` should be parameterized to handle the product of `s3_folder, gcs_folder, azure...` to `s3, gcs, etc` and therefore include both `s3_and_s3` and `gcs_and_gcs` options

```
def test_s3_and_s3(local_folder, s3_folder):
    # from one s3 folder to another s3 folder
    new_s3_folder = s3_folder.to(system="s3")
    assert "sample_file_0.txt" in new_s3_folder.ls(full_paths=False)

    new_s3_folder.rm()
```